src/core/util/env.js

```diff
@@ -16,6 +16,7 @@ export const isIOS = (UA && /iphone|ipad|ipod|ios/.test(UA)) || (weexPlatform ==
 export const isChrome = UA && /chrome\/\d+/.test(UA) && !isEdge
 export const isPhantomJS = UA && /phantomjs/.test(UA)
 export const isFF = UA && UA.match(/firefox\/(\d+)/)
+export const isCEP = inBrowser && window.__adobe_cep__ !== undefined
```

Comment:
I think checking `window.__adobe_cep__` is enough since we know `window` is defined

Reply:
Can you clarify? Which of the following do you mean:

1. ```js
    export const isCEP = window.__adobe_cep__ !== undefined
    ```
2. ```js
    export const isCEP = inBrowser && window.__adobe_cep__
    ```

The first would cause issues where `window` is not defined (i.e. not in `inBrowser` contexts) while the second seems a bit ambiguous and doesn't follow the `undefined` check style of other options.

Or did you mean something  else?

Reply:
I spoke with @posva in a separate forum (Discord direct chat) and it was determined that using the `isObject` utility function was the best approach to validating `window.__adobe_cep__`.